api/routers/cameras.py

```diff
@@ -236,6 +239,7 @@ async def get_camera_image(camera_id: str):
     """
     Gets the image related to the camera <camera_id>
     """
+    _ = get_camera_from_id(camera_id)
```

Comment:
While this probably works, it doesn't sound right. I think you are misusing the function or at least the name is very misleading